examples/preprocessing/plot_muscle_detection.py

```diff
@@ -0,0 +1,92 @@
+"""
+===========================
+Annotate muscle artifacts
+===========================
+
+Muscle contractions produce high frequency activity that can mask brain signal
+of interest. Muscle artifacts can be produced when clenching the jaw,
+swallowing, or twitching a head muscle. Muscle artifacts are most notable
+in the range of 110-140 Hz.
+
+This example uses :func:`~mne.preprocessing.annotate_muscle` to annotate
+segments where muscle activity is likely present. This is done by band-pass
+filtering the data in the 110-140 Hz range. Then, the envelope is taken using
+the hilbert analytical signal to only consider the absolute amplitude and not
+the phase of the high frequency signal. The envelope is z-scored and averaged
+across channels. To remove noisy transient peaks (muscle artifacts last
+several hundred milliseconds), the channel averaged z-scored is low-pass
+filtered to 4 Hz. Segments above a set threshold are annotated as
+``BAD_muscle``. In addition, ``min_length_good`` parameter determines the
+cutoff for whether short spans of "good data" in between muscle artifacts are
+included in the surrounding "BAD" annotation.
+
+
+.. note::
+    If line noise is present, you should perform notch-filtering *before*
+    detecting muscle artifacts. See :ref:`tut-section-line-noise` for an
+    example.
+
+"""
+# Authors: Adonay Nunes <adonay.s.nunes@gmail.com>
+#          Luke Bloy <luke.bloy@gmail.com>
+# License: BSD (3-clause)
+
+import os.path as op
+import matplotlib.pyplot as plt
+from numpy import arange
```

Comment:
we don't do this. The convention is to do

import numpy as np
np.arange

thanks